playbooks/luna.yml

```diff
@@ -0,0 +1,19 @@
+---
+- import_playbook: katello.yml
+  vars:
+    foreman_installer_additional_packages:
+      - katello
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
+      - "--enable-foreman-plugin-ansible"
+      - "--enable-foreman-plugin-discovery"
+      - "--enable-foreman-plugin-foreman-bootdisk"
+      - "--enable-foreman-plugin-foreman-hooks"
+      - "--enable-foreman-plugin-foreman-openscap"
+      - "--enable-foreman-plugin-foreman-templates"
+      - "--enable-foreman-plugin-remote-execution"
+      - "--enable-foreman-proxy-plugin-remote-execution-ssh"
```

Comment:
Would it make sense to include these proxy plugins in the devel scenario as well?